services/QuillLMS/app/controllers/pages_controller.rb

```diff
@@ -514,6 +514,12 @@ def administrator
     @title = 'Administrator'
   end
 
+  def locker
+    redirect_to(profile_path) && return if !current_user || (current_user && current_user.role != 'staff')
```

Comment:
I think we have an auth method for this so you could just do `return redirect_to profile_path if !staff?`

Reply:
nice, thanks for pointing that out! I'll update